src/v2/view-builder/views/IdentifyRecoveryView.js

```diff
@@ -12,6 +12,18 @@ const Body = BaseForm.extend({
     return loc('oform.next', 'login');
   },
 
+  initialize() {
+    BaseForm.prototype.initialize.apply(this, arguments);
+    const identifier = this.options.appState.get('lastIdentifier');
+    if (identifier) {
+      this.model.set('identifier', identifier);
+      // Toggle Form saving status (e.g. disabling save button, etc)
+      this.model.trigger('request');
+      // Auto submit
+      this.trigger('save', this.model);
```

Comment:
Auto-fill identifier and trigger save to fix monolith tests.

**Auto save looks like a hack**
Alternatives:
- change monolith tests by adding click on "Next" button
- hide reset pass form until we got response on saving

Reply:
Curious why monolith tests fails after this change? caused by transaction state change, like without failed transaction in state, the form also lost identifier from UI perspective?

Reply:
Monolith tests are not failing if these line 23 (auto-submit) is added:
```
      this.trigger('save', this.model);
```

Eg. see test `testPasswordExpiredWhenResetDisabled` at `IdxADPasswordLifecycle2IT.java`.
It expects that after clicking 'Forgot password' user sees message that reset password is not allowed.

Reply:
https://github.com/okta/okta-signin-widget/assets/72614880/36007b68-bb92-4870-84df-9240edb97ede

Reply:
> without failed transaction in state, the form also lost identifier from UI perspective?
Yes, that's why I had to save identifier to appState (with key `lastIdentifier `)